README.md

```diff
@@ -0,0 +1,76 @@
+﻿# Interactive Hall Display
+
+Welcome! This is the source code repository for the Interactive Hall Display located outside James Madison University's Computer Science department.
+
+This display provides updates, announcements, and events to the student body throughout the semester. Students are welcome to fork this repository and submit pull-requests to add functionality to the display. 
+
+If you have questions about the contents of this repository, please contact `bowersjc@jmu.edu` or an executive member of the JMU Game Development club.
+
+Game Development Executives for Fall 2018:
+
+President: Jason Zareski -- <zareskj@dukes.jmu.edu>\
```

Comment:
Email should be, zareskjj@dukes.jmu.edu.

Reply:
Change made in this commit:
https://github.com/JMU-CS/Interactive-Hall-Display/pull/7/commits/e6776c6689a002f082521e7ceecdcaccddb18d28